x-wing/Cargo.toml

```
@@ -0,0 +1,35 @@
+[package]
+name = "x-wing"
+description = """
+Pure Rust implementation of the X-Wing Key-Encapsulation Mechanism
+"""
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.81"
+license = "Apache-2.0 OR MIT"
+readme = "README.md"
+homepage = "https://github.com/RustCrypto/KEMs/tree/master/ml-kem"
+repository = "https://github.com/RustCrypto/KEMs"
+categories = ["cryptography", "no-std"]
+keywords = ["crypto", "x-wing", "xwing", "kem", "post-quantum"]
+
+[dependencies]
+rand_core = { version = "0.6.4", default-features = false }
+x25519-dalek = { version = "2.0", features = [
+    "static_secrets",
+    "reusable_secrets",
+] }
+ml-kem = { version = "*", features = ["deterministic"] }
```

```suggestion
ml-kem = { version = "0.2.1", features = ["deterministic"], path = "../ml-kem" }
```

...will ensure that breaking changes to `ml-kem` also receive corresponding changes in `x-wing`